sapi/phpdbg/phpdbg_prompt.c

```diff
@@ -1675,7 +1675,7 @@ static inline void list_code() {
 					EG(exception) = exception; \
 				} else { \
 					Z_OBJ(zv) = exception; \
-					zend_throw_exception_internal(&zv); \
+					zend_throw_exception_internal(Z_OBJ(zv)); \
```

Comment:
```suggestion
					zend_throw_exception_internal(exception); \
```
and drop the `zv` variable, it was just a workaround for exactly this issue :)